bundles/prepend_extension.rst

```diff
@@ -186,6 +181,12 @@ method::
 
     The ``prependExtension()`` method, like ``prepend()``, is called only at compile time.
 
+.. versionadded:: 7.1
```

Comment:
Isn't it `.. deprecated::` instead? It's not really a deprecation, but not something "added" 🤔

Reply:
Fixed, thanks!

Reply:
Although it's not really a deprecation, it is an acceptable BC behavior break. @javiereguiluz, do we have any other mark/syntax for these cases?

Reply:
well, at this point I'm not even sure that `deprecated` is the best option. added feels right to me: the behavior change was added in 7.1

Reply:
I think `deprecated` is safer because as a dev, I personally think I would definitely pay more attention to a `deprecated` block than a `versionadded` one 😄